src/android/FirebasePlugin.java

```diff
@@ -795,6 +799,12 @@ public void onCodeSent(String verificationId, PhoneAuthProvider.ForceResendingTo
             }
         });
     }
+	
+    private String getPrivateField(PhoneAuthCredential credential, String field) throws NoSuchFieldException, IllegalAccessException {
```

Comment:
Really minor, but since this does not have any class dependencies, it's better off being a static method.

Reply:
changed